### ISSUE TITLE
Android Running at 13-15 FPS even on high end device.

### ISSUE THREAD

**Issue by sanchitgng** [Unity-Technologies/BoatAttack#93]
I tried to find out the issue using unity profiler tools but it is showing like it's both CPU and GPU bound, that can't be the case.
I think it's a problem with the URP, I tried opening it in various 2019.3 unity versions but the result was same, tried tweaking various quality settings, texture resolution and almost everything but all in vain. 
Even ran the system observer on my android and found that the CPU and GPU was 30% used at the time of running application(running at 15 fps).

Below is an image of profiler getting data from my android (Xiaomi Mi 9/ Redmi K20).

<img width="639" alt="BoatAttackProfiler" src="https://user-images.githubusercontent.com/40668553/81177242-f630a800-8fc3-11ea-9171-25f01cbe40bb.PNG">

**Comment by sanchitgng** [comment]
After digging more into the issue I found out that Vulkan API is causing some of the problems, switching to OpenGLES doubles the frame rates and now it runs at 30 fps, I am still aiming to somehow make it to 60fps. Let's see if it happens.

**Comment by StinkySteak** [comment]
Yes, I can guess you are using Android 10 at that time. Android 10 Vulkan is very faulty. also happens to my samsung a70. upgraded to android 11 now it 75% faster.
From g3dmark, your xiaomi m9, from android 10 to android 11, it has a score from 2086 to 3333 (wild life benchmark)

Lesson learnt: always use OpenGL if you want to accomodate low-end device
